test/tests/functional/pbs_multi_sched.py

```diff
@@ -544,11 +550,28 @@ def test_job_sorted_per_scheduler(self):
                             {'scheduling': 'True'}, id="sc3")
         self.server.expect(JOB, {'job_state': 'R'}, id=jid4)
 
+    def test_qrun_job(self):
+        """
+        Test jobs are sorted as per job_sort_formula
+        inside each scheduler
```

Comment:
How does this test the job_sort_formula?  There is only one job.

Reply:
I will change the comment , It is test of qrun.